backend/infrastructures/extract/extract.py

```diff
@@ -0,0 +1,30 @@
+import modules.extract.extract  as ex
+from pathlib import Path
+import logging
+import pandas as pd
+import infrastructures.extract.strategies as strat
+
+
+class ExtractManager:
```

Comment:
Extract manager should be placed in domain logic.

Reply:
I had doubts to this point, but I think you're correct, it should be domain. Let's break it down.
Business requirement is to extract .csv, excel and pdf files. Although domain doesn't care how it is done. 
As long as ExtractManager's job is to choose a strategy and execture it's "extract" method, it doesn't contain any implementation details. It just represents the business case. So, I agree it should be in domain.

Reply:
Exactly.